snowflake_data_profiler/routes/default.py

```diff
@@ -26,7 +26,8 @@ def post_profile():
         database  = req.get('database')
         schema    = req.get('schema')
         table     = req.get('table')
-        pd_df = connect_to_snowflake(username, password, url, database, schema, table, warehouse, role)
+        connector = establish_connection(username, password, url, database, schema, table, role)
+        pd_df = create_cursor(connector, database, schema, table, warehouse)
```

Comment:
prefer to use the `main` function in the profiler. Try to abstract the details of this if possible to a main handler